resolwe/storage/tests/test_listener.py

```diff
@@ -0,0 +1,84 @@
+# pylint: disable=missing-docstring
+from unittest.mock import MagicMock, patch
+
+from resolwe.flow.managers.listener import ExecutorListener
+from resolwe.flow.managers.protocol import ExecutorProtocol
+from resolwe.storage.models import FileStorage, ReferencedPath, StorageLocation
+from resolwe.test import TestCase
+
+
+class ListenerTest(TestCase):
+    def setUp(self):
+        self.listener = ExecutorListener()
+        self.file_storage = FileStorage()
```

Comment:
Add a new line above to separate infrastructure and data setup.

Reply:
OK.